forte/mrdsrg-spin-integrated/dsrg_mrpt2_grad/dsrg_mrpt2_deriv_setb.cc

```diff
@@ -0,0 +1,1091 @@
+                temp_1["vx"] -= V_["vExZ"] * T2_["uWyE"] * Gamma1_["ZW"] * Gamma1_["yu"];
+                temp_1["uy"] -= V_["vExZ"] * T2_["uWyE"] * Gamma1_["ZW"] * Gamma1_["xv"];
+                temp_1["xu"] -= V_["vExZ"] * T2_["uWyE"] * Gamma1_["yv"] * Gamma1_["ZW"];
+                temp_1["yv"] -= V_["vExZ"] * T2_["uWyE"] * Gamma1_["xu"] * Gamma1_["ZW"];
+                temp_1["xv"] += V_["vExZ"] * T2_["uWyE"] * Gamma1_["yu"] * Gamma1_["ZW"];
+                temp_1["yu"] += V_["vExZ"] * T2_["uWyE"] * Gamma1_["xv"] * Gamma1_["ZW"];
+
+                // -0.50 * V_.block("aVaA")("uExY") * T2_.block("AAVA")("VWEZ") *
+                // dlamb3_abb("KxYZuVW")
+                temp_1["xu"] += 0.50 * V_["uExY"] * T2_["VWEZ"] * Lambda2_["YZVW"];
+                temp_1["ux"] -= 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["ux"] += 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["YW"] * Gamma1_["ZV"];
+                temp_1["ux"] += 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["ZV"] * Gamma1_["YW"];
+                temp_1["ux"] -= 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["ZW"] * Gamma1_["YV"];
+                temp_1["xu"] += 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["xu"] -= 0.50 * V_["uExY"] * T2_["VWEZ"] * Gamma1_["ZV"] * Gamma1_["YW"];
+
+                // 0.50 * V_.block("AVAA")("WEYZ") * T2_.block("aAaV")("uVxE") *
+                // dlamb3_abb("KxYZuVW")
+                temp_1["xu"] -= 0.50 * V_["WEYZ"] * T2_["uVxE"] * Lambda2_["YZVW"];
+                temp_1["ux"] += 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["ux"] -= 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["YW"] * Gamma1_["ZV"];
+                temp_1["ux"] -= 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["ZV"] * Gamma1_["YW"];
+                temp_1["ux"] += 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["ZW"] * Gamma1_["YV"];
+                temp_1["xu"] -= 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["xu"] += 0.50 * V_["WEYZ"] * T2_["uVxE"] * Gamma1_["ZV"] * Gamma1_["YW"];
+
+                // V_.block("vAaA")("eWxY") * T2_.block("aAvA")("uVeZ") * dlamb3_abb("KxYZuVW")
+                temp_1["xu"] -= V_["eWxY"] * T2_["uVeZ"] * Lambda2_["YZVW"];
+                temp_1["ux"] += V_["eWxY"] * T2_["uVeZ"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["ux"] -= V_["eWxY"] * T2_["uVeZ"] * Gamma1_["YW"] * Gamma1_["ZV"];
+                temp_1["ux"] -= V_["eWxY"] * T2_["uVeZ"] * Gamma1_["ZV"] * Gamma1_["YW"];
+                temp_1["ux"] += V_["eWxY"] * T2_["uVeZ"] * Gamma1_["ZW"] * Gamma1_["YV"];
+                temp_1["xu"] -= V_["eWxY"] * T2_["uVeZ"] * Gamma1_["YV"] * Gamma1_["ZW"];
+                temp_1["xu"] += V_["eWxY"] * T2_["uVeZ"] * Gamma1_["ZV"] * Gamma1_["YW"];
+            }
+
+            // - H.block("aa")("vu") * cc1a("Kuv")
```

Comment:
There is some code commented out around here. Please remove, especially if incorrect or less efficient.